packages/manager/modules/freefax/src/freefaxes.controller.js

```diff
@@ -0,0 +1,23 @@
+import { ListLayoutHelper } from '@ovh-ux/ng-ovh-telecom-universe-components';
+
+export default class FreefaxesCtrl extends ListLayoutHelper.ListLayoutCtrl {
+  /* @ngInject */
+  constructor($q, ouiDatagridService) {
```

Comment:
Both `$q` and `ouiDatagridService` are really needed?

I can't find any references.